trade-finance-manager-api/src/v1/services/deal-cancellation/deal-cancellation.service.ts

```diff
@@ -61,7 +61,7 @@ export class DealCancellationService {
   public static async submitDealCancellation({ dealId, cancellation, auditDetails }: SubmitDealCancellationParams) {
     console.info(`Submitting deal cancellation for ${dealId}`);
 
-    const { cancelledDealUkefId, riskExpiredFacilityUkefIds } = await api.submitDealCancellation({ dealId, cancellation, auditDetails });
+    const { cancelledDeal, riskExpiredFacilityUkefIds } = await api.submitDealCancellation({ dealId, cancellation, auditDetails });
```

Comment:
Is there any reason we updated this returning a status to now return a whole deal?

Reply:
I added a comment but think its gone out of date & disappeared into the ether:

I've realised I need more information about the deals in the service (the dealType as well as ukefDealId) - I think it makes sense to return the whole deal object. This should keep it adaptable for the future too

Reply:
We're going to leave this for now as there is already existing patterns in dtfs-tfmapi that pass the whole object. In future, it would be good to see the minimum amount passed between B.E services please:)